packages/components/src/font-size-picker/test/index.tsx

```diff
@@ -395,7 +410,12 @@ describe( 'FontSizePicker', () => {
 					screen.getByRole( 'radio', { name: radio } )
 				);
 				expect( onChange ).toHaveBeenCalledTimes( 1 );
-				expect( onChange ).toHaveBeenCalledWith( expectedValue );
+				expect( onChange ).toHaveBeenCalledWith(
+					expectedValue,
+					fontSizes.find(
```

Comment:
@noisysocks Did I just butcher your tests? 😄 

Would you think it's better to be explicit about expected mock args here like I've done [above](https://github.com/WordPress/gutenberg/pull/44967/files#diff-e9acdac412fe0b8c73464e6678c0c58d73f01a10d175ff0ec33bbb4bef3ba9c2R199)?

`expect( onChange.mock.calls ).toEqual( [ expectedValue ]`

Reply:
I personally prefer an explicit assertion so that the tests don't become an exact replica of the code being tested. 

You might also use `toHaveBeenCalledWith( ...expectedArguments )` instead of `mock.calls` because reads a little more like English. 

Doesn't really matter though. Developer preference!

Reply:
Good call, I like! Will update, thanks.